### ISSUE TITLE
Find a way to use boxplots with time

### ISSUE THREAD

**Issue by guidocioni** [guidocioni/point_wx#32]

**Comment by guidocioni** [comment]
Finally found a workaround thanks to ChatGPT.
The idea is that we have to manually place every boxplot (duh) and multiply the index by the columns length...Don't know why but it works.

```python
# Create a single Plotly figure
fig = go.Figure()

# Add box plots for each time instance
for index, row in df.set_index('time').iterrows():
    fig.add_trace(go.Box(
        x=[index] * len(row),
        y=row,
        showlegend=False,
        boxpoints=False,
        marker_color='gray',
    ))
```

Note that, in this case, the xaxis is a datetime so we could easily plot a scatter (or everything else) on top of it.

Now I have to implement it in the app to see if it looks good or not.

**Comment by guidocioni** [comment]
Implemented in 36114e56242a1b1e91acedba03599b651758dce9.
Not sure if it is more readable than having individual lines
